<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>461</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>201</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiarTexto">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrirArchivo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir Archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrirArchivo_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Tokenizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerarHTML">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar HTML</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTituloCampotexto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Documento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estructura de Listas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
